sphinx_rtd_theme/footer.html

```diff
@@ -48,13 +48,16 @@
 
   {% if show_sphinx %}
     {%- set sphinx_web = '<a href="https://www.sphinx-doc.org/">Sphinx</a>' %}
-    {%- set readthedocs_web = '<a href="https://readthedocs.org">Read the Docs</a>'  %}
     {#- Translators: the variable "sphinx_web" is a link to the Sphinx project documentation with the text "Sphinx" #}
     {%- trans sphinx_web=sphinx_web, readthedocs_web=readthedocs_web %}Built with {{ sphinx_web }} using a{% endtrans %}
     {#- Translators: "theme" refers to a theme for Sphinx, which alters the appearance of the generated documentation #}
     <a href="https://github.com/readthedocs/sphinx_rtd_theme">{% trans %}theme{% endtrans %}</a>
     {#- Translators: this is always used as "provided by Read the Docs", and should not imply Read the Docs is an author of the generated documentation. #}
-    {% trans %}provided by {{ readthedocs_web }}{% endtrans %}.
+    {% trans %}provided by Read the Docs{% endtrans %}.
+  {% endif %}
+
+  {% if READTHEDOCS %}
+    Hosted on <a href="https://about.readthedocs.com/">Read the Docs</a>.
```

Comment:
This should probably be wrapped by `show_sphinx` or a specific configuration option. If the user selected `show_sphinx = False`, they will probably not like that we injected different content in the same space.